packages/client/src/pages/Game/components/Footer/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+import './styles.scss';
+
+export const Footer: React.FC = () => {
+  return (
+    <div className="footer">
+      <span className="footer_title">Как играть</span>
+      <br/>
+      <span className="footer_text">
```

Comment:
Раз уж решили наименовать классы самостоятельно, давайте лучше используем [БЭМ](https://ru.bem.info/methodology/), тут правильнее будет:
```suggestion
      <span className="footer__text">
```
блок__элемент